### PR TITLE
NO-REF update readme for clarity

### PR DESCRIPTION
**Pull request by jrschumacher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,98 @@
...
+$ cd encrypt-decrypt-demo
+
+# Install node via NVM
+$ nvm use
+
+# Install node modules
+$ npm ci
+
+# Start the node server
+$ sudo npm start
```
Comment (anelsonvirtru) [inline README.md]: we don't need sudo to start this demo

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Simple example to encrypt and decrypt files in the TDF spec using Virtru hosted KAS and EAS.",
   "scripts": {
     "build": ". simple/scripts/build.sh",
-    "setup": "npm install; npm run build",
+    "prestart": "npm run build",
```
Comment (anelsonvirtru) [inline package.json]: 👍

File: README.md (hunk 1 above)
Comment (jrschumacher) [reply README.md]: Nice okay

**Review by anelsonvirtru — DISMISSED** [review]

**Review by anelsonvirtru — APPROVED** [review]
LGTM

**Pull request merged**